backend/src/main/java/com/google/models/Anomaly.java

```diff
@@ -135,7 +136,7 @@ public static Anomaly createAnomalyFromEmbeddedEntity(EmbeddedEntity anomalyEmbe
 
     if (dataPointsEE != null) {
       for (String key : dataPointsEE.getProperties().keySet()) {
-        dataPointsMap.put(new Timestamp(key), new MetricValue((int) dataPointsEE.getProperty(key)));
+        dataPointsMap.put(new Timestamp(key), new MetricValue(((Long) dataPointsEE.getProperty(key))));
```

Comment:
btw, is there a reason why this one is casted into the `Long` object whereas the line 118 above is casted into the primitive `long` type?

Reply:
The reason that in line 118 `(long) metricValue.getValue()`, the value is casted to a primitive instead of a `Long` object is that `getValue()` will always return an `int` and will not return `null`. 

But in the case of line 139 above, `dataPointsEE.getProperty` could return a `null` object and if that is casted into a primitive instead of a `Long` object, then it will result in a `NullPointerException`.

But now that I think about it, if `dataPointsEE.getProperty` returns a `null` object and that gets passed into `MetricValue` constructor, it would still result in an exception....

I guess it's better to keep the two consistent with both casting into primitives in this case?